kafka-1.0.x/src/main/scala/monix/kafka/KafkaConsumerConfig.scala

```diff
@@ -428,7 +429,7 @@ object KafkaConsumerConfig {
       retryBackoffTime = config.getInt("retry.backoff.ms").millis,
       observableCommitType = ObservableCommitType(config.getString("monix.observable.commit.type")),
       observableCommitOrder = ObservableCommitOrder(config.getString("monix.observable.commit.order")),
-      observableSeekToEndOnStart = config.getBoolean("monix.observable.seekEnd.onStart"),
+      observableStartFrom = if (config.getBoolean("monix.observable.seekEnd.onStart")) FromLatest else FromCommited,
```

Comment:
We could do something similar to `ObservableCommitOrder` and `ObservableCommitType` and create `apply` method.
We could also consider allowing all the options using `conf` file (probably with two parameters)

Reply:
I actually think these are things that should be configured on a per consumer basis, maybe even on a topic level.

Reply:
Ok, makes sense. :)